src/hist/basehist.py

```diff
@@ -87,37 +87,37 @@ def __init__(
             for a in args
         ]
 
-        if args:
```

Comment:
I just completely removed this if, wasn't needed, and it caused the `super().__init__` to never get called.

Reply:
(If you hide whitespace in the GitHub diff, you'll see basically nothing changed really here except this line)